specification/implementation/structure/implementation/index.md

```diff
@@ -2,10 +2,12 @@
 title: Implementing the structure
 ---
 ## 4.2.	Implementing the structure
-The requirements presented in Section 4.1 leave room for quite a few decisions during implementation. For the sake of clarity we provide here examples for two extremes – the simplest and the full use of the structure.
+The requirements presented in Section 4.1 leave room for decisions during the implementation. For the sake of clarity we here provide examples for two extremes – the simplest and the full use of the structure.
```

Comment:
we here provide examples
to
we provide examples

Reply:
Updated